src/app/shared/components/banner/banner.component.html

```diff
@@ -0,0 +1,19 @@
+<ecl-banner
+    [variant]="variant"
+    isCentered="{{isCentered }}"
+    >
+    <picture eclBannerPicture *ngIf="imageUrl && variant != 'plain-background'">
+      <img eclBannerImage src="{{ imageUrl }}" alt="alternative text" />
+    </picture>
+  <div eclBannerTitle *ngIf="title">
+    <span eclBannerTitleText>{{ title }}</span>
+  </div>
+  <div eclBannerCredit *ngIf="credit">{{ credit }}</div>
+  <p eclBannerDescription>
+    <span eclBannerDescriptionText>{{ description }}</span>
+  </p>
+  <a eclLink routerLink="/" variant="cta">
```

Comment:
why did you let this harcoded here? Should be like the others depending on an input.

Reply:
yes, changed it to input.